stylesheets/_touch-friendly-links.scss

```diff
@@ -0,0 +1,10 @@
+// This mixin will expand the clickable area for a link on mobile and tablet to make it easier to click on a touch
+// device.
+
+@mixin touch-friendly-links($padding: 3px) {
+  a {
+    padding: $padding;
+    margin: -1 * $padding;
+    outline: none;
```

Comment:
Does removing the outline cause any issues identifying the focussed element when the user has overridden text / background colours in their browser?